db/migrate/20180910155506_add_uniqueness_of_order_id_to_spree_shipments.rb

```diff
@@ -1,6 +1,9 @@
 # This migration is an OFN specific migration that enforces an order to have a single shipment at all times
 class AddUniquenessOfOrderIdToSpreeShipments < ActiveRecord::Migration
   def change
+    Spree::InventoryUnit.connection.schema_cache.clear!
+    Spree::InventoryUnit.reset_column_information
```

Comment:
I don't think it makes any difference in terms of behavior but I find more intention reveling to use `ActiveRecord::Base` in this cases, so the next dev doesn't think it's something related to `InventoryUnit`.

Reply:
It _is_ related to `InventoryUnit`. From my comment in the issue:

> It's failing with missing attribute: lock_version at the call to shipments_to_delete.destroy_all, and I'm pretty sure this is caused by the deletion of related objects with dependent: destroy, namely Spree::InventoryUnit, which has it's lock_version attribute removed in a migration a few steps before this one.

Reply:
I could add a comment?